README.md

```diff
@@ -2,29 +2,84 @@
 
 Product focuses on 100% education as well as upskilling developing countries and rural areas. 
 
-## Available Scripts
+# How To Run the Project
 
-In the project directory, you can run:
+***1.*** Open Project Folder</br>
 
-### `yarn start`
+***2.*** To install all dependencies run
+```
+yarn install
+```
+***3.*** To run development server.This will run the development server in the http://localhost:3000
+```
+yarn start
+```
 
```

Comment:
Also add commands for npm,  you can use collapsible for that: https://gist.github.com/joyrexus/16041f2426450e73f5df9391f7f7ae5f

Some people previously had problem starting with yarn, since they use npm, better we can provide both in collapsible menu :)